src/super_gradients/training/metrics/detection_metrics.py

```diff
@@ -30,6 +30,7 @@ class DetectionMetrics(Metric):
     :param dist_sync_on_step:               Synchronize metric state across processes at each ``forward()`` before returning the value at the step.
     :param accumulate_on_cpu:               Run on CPU regardless of device used in other parts.
                                             This is to avoid "CUDA out of memory" that might happen on GPU.
+    :param calc_best_score_thresholds       Whether to calculate the best score threshold per class
```

Comment:
If we leave it this way, please give some guidance what we will have retiurned when we use this flag.